lib/traject/config/sirsi_config.rb

```diff
@@ -13,6 +14,24 @@
   provide "reader_class_name", "Traject::MarcCombiningReader"
 end
 
+RESERVES_FILE = ENV['RESERVES_FILE'] || './spec/fixtures/files/multmult.csv'
```

Comment:
I wonder if this can be a traject setting instead of an env variable directly? That might make it easier to test (and, I assume, mean we won't need to hard-code a spec-path location here?)

Reply:
k. what do you think about this approach?